### DISCUSSION TITLE
How to express 'severity' modifiers of a MAxO term

### DISCUSSION THREAD

**Discussion by LCCarmody** [monarch-initiative/MAxO#349] [General]
Trying to express the severity of a MAxO term.  There are two sides of the coin. The most commonly expressed way is to say to 'Avoid excessive X', but could be 'Low/moderate X'.

Possible way to express it:
equivalentTo:
  text: "'nutrition intervention' and ('avoided_food' some %s) and ('has_modifier' some %s))"
  vars:
    - food_material
    - severity

Added avoidance_of_food_intake_by_severity.yaml pattern (on behavior avoidance branch).

Use cases:
Low sodium diet intake/Avoid (excessive) salt intake
Low protein intake/Avoid excessive protein
Moderate exercise/Avoid excessive exercise
Avoid morbid obesity
Avoid excessive sun exposure
Avoid excessive muscle exercise, avoid intense isometric exercise,  avoid maximal aerobic exercise
Avoid excessive weight loss
Avoid excessive alcohol intake
Avoid excessive fasting or avoid excessive caloric deprivation/Avoid prolonged fasting
Maintenance of body temperature/Avoid high body temperature
Avoid prolonged immobility
prophylactic antiemetics (possibly)
"Excess protein intake; less than recommended intake of protein" recommendation of 'moderate protein intake'?
avoid excessive heat or cold
avoid excessive sound
avoid irregular sleep

Too many others.

**Comment by LCCarmody** [comment]
@matentzn

**Comment by LCCarmody** [comment]
- [ ] avoid excess nitrogen consumption
- [ ] high salt diet (maybe a diet)
- [ ] avoid excess vitamin D intake
- [ ] avoid extreme diets (this may not be appropriate here because you would not have a term to 'avoid diets') #112

**Comment by matentzn** [comment]
The arguments here are the same as for phenotypes. Rarely is it a good idea to compose regular classes with severity modifiers if we want to avoid combinatorial explosions. I would need to question the specific use case at hand.

**Comment by pnrobinson** [comment]
Probably it is simpler to treat Avoid excessive sun exposure and Avoid sun exposure the same, since "excessive" is a subjective thing and one would not advise patients to "get some but not excessive exposure to X" since that would be impossible to quantify...

**Comment by LCCarmody** [comment]
Good to know. Will close this and write a note in the read me file for how to make such terms. Thx.
